src/Admin/AbstractAdmin.php

```diff
@@ -1655,7 +1655,7 @@ final public function supportsPreviewMode(): bool
     /**
      * Returns predefined per page options.
      *
-     * @return list<int>
+     * @return array<int>
```

Comment:
This is the only one I have doubts, why we were using list? Phpstan complains with:

```
 ------ --------------------------------------------------------------------------------------------------------
  Line   src/Admin/AbstractAdmin.php
 ------ --------------------------------------------------------------------------------------------------------
  1672   Method Sonata\AdminBundle\Admin\AbstractAdmin::getPerPageOptions() should return list<int> but returns
         non-empty-array<0|1|2|3|4|5, int>.
```

Reply:
Only usage is `{% for per_page in admin.getperpageoptions %}` and
`return \in_array($perPage, $this->getPerPageOptions(), true);`
So it doesn't require to be a list.

I have one more question:
Why this method is not in the interface but is public and used in the twig ?

Can you add it (with @method) ?

Reply:
Anyway can be done in another pR